src/components/redux/books/books.js

```diff
@@ -0,0 +1,20 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const reducer = (initialState = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
```

Comment:
- [ ] This reducer should work as expected, unfortunately, when I tried to add or delete a book I failed to do that. Depending on the action type a user should be able to delete or add a book.